packages/components/src/focal-point-picker/README.md

```diff
@@ -94,3 +94,10 @@ Callback which is called at the end of drag operations.
 -   Required: No
 
 Callback which is called at the start of drag operations.
+
+### `resolvePoint`
+
+-   Type: `Function`
+-   Required: No
+
+A function that receives the current value and returns a modified value. It is called before the onDrag callback.
```

Comment:
The last sentence is outdated since 54d8cc0b92445a5fdbf5762789dce16f8617ccb0. Also I think it might be improved worded more like this:
```suggestion
Function which is called before internal updates to the value state. It receives the upcoming value and may return a modified one.
```